app/graphql/types/mutation_type.rb

```diff
@@ -14,6 +15,7 @@ def create_user(**_args)
         name: _args[:name],
         email: _args[:email],
         password: _args[:password],
+        password_confirmation: _args[:password_confirmation],
```

Comment:
_ (dash) at the beginning of a variable name suggests that it won't be used in method body - its a common ruby convention. _args need to be changed to args here as it's misleading right now